cmd/mesh-bridge/main.go

```diff
@@ -0,0 +1,112 @@
+// +build !js
+
+// mesh-bridge is a short program that bridges two Mesh nodes. This is useful in cases where
+// we introduce a network-level breaking change but still want the liquidity from one network
+//to flow to another
+package main
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/0xProject/0x-mesh/rpc"
+	"github.com/0xProject/0x-mesh/zeroex"
+	ethrpc "github.com/ethereum/go-ethereum/rpc"
+	"github.com/plaid/go-envvar/envvar"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	firstWSRPCAddressLabel  = "FirstWSRPCAddress"
+	secondWSRPCAddressLabel = "SecondWSRPCAddress"
+	maxReceiveBatch         = 10
+	receiveTimeout          = 10 * time.Second
+)
+
+type clientEnvVars struct {
+	FirstWSRPCAddress  string `envvar:"FIRST_WS_RPC_ADDRESS"`
+	SecondWSRPCAddress string `envvar:"SECOND_WS_RPC_ADDRESS"`
+	Verbosity          int    `envvar:"VERBOSE"`
+}
+
+func main() {
+	env := clientEnvVars{}
+	if err := envvar.Parse(&env); err != nil {
+		panic(err)
+	}
+
+	log.SetLevel(log.Level(env.Verbosity))
+
+	firstClient, err := rpc.NewClient(env.FirstWSRPCAddress)
+	if err != nil {
+		log.WithError(err).Fatal("could not create client")
+	}
+	stats, err := firstClient.GetStats()
+	if err != nil {
+		log.Fatal(err)
+	}
+	log.WithField("stats", stats).Info("Spun up first client")
+
+	secondClient, err := rpc.NewClient(env.SecondWSRPCAddress)
+	if err != nil {
+		log.WithError(err).Fatal("could not create client")
+	}
+	stats, err = secondClient.GetStats()
+	if err != nil {
+		log.Fatal(err)
+	}
+	log.WithField("stats", stats).Info("Spun up second client")
+
+	go func() {
+		pipeOrders(secondClient, firstClient, secondWSRPCAddressLabel, firstWSRPCAddressLabel)
+	}()
+	pipeOrders(firstClient, secondClient, firstWSRPCAddressLabel, secondWSRPCAddressLabel)
+}
+
+func pipeOrders(inClient, outClient *rpc.Client, inLabel, outLabel string) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	orderEventsChan := make(chan []*zeroex.OrderEvent, 8000)
```

Comment:
Why 8000? Can we make this a constant?

Reply:
This was copied over from `subscribe-to-orders` example. Moved to constant.